sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1089,7 +1089,8 @@ _pi_queue::getZeCopyCommandQueue(int *CopyQueueIndex,
   int n = ZeCopyCommandQueues.size();
   int LowerCopyQueueIndex = getRangeOfAllowedCopyEngines.first;
   int UpperCopyQueueIndex = getRangeOfAllowedCopyEngines.second;
-  LowerCopyQueueIndex = std::max(0, LowerCopyQueueIndex);
+  LowerCopyQueueIndex =
+      (LowerCopyQueueIndex == -1) ? -1 : std::max(0, LowerCopyQueueIndex);
```

Comment:
if your range is [-1:-1] then the check at line 1098 is not going to evaluate to "true"